src/content/docs/fr/guides/integrations-guide/cloudflare.mdx

```diff
@@ -0,0 +1,403 @@
+
+```ts
+// astro.config.mjs
+export default defineConfig({
+  adapter: cloudflare({ mode: 'directory' }),
+});
+```
+
+En mode `directory`, l'adaptateur compilera la partie côté client de votre application de la même manière qu'en mode `advanced` par défaut, mais déplacera le script worker dans un dossier `functions` à la racine du projet. Dans ce cas, l'adaptateur placera un `[[chemin]].js` dans ce dossier, ce qui vous permet d'ajouter des plugins et des pages middleware supplémentaires qui peuvent être vérifiés dans le contrôle de version.
+
+Pour compiler un paquet séparé pour chaque page, définissez l'option `functionPerPath` dans la configuration de votre adaptateur Cloudflare. Cette option nécessite une maintenance manuelle du dossier `functions`. Les fichiers émis par Astro écraseront les fichiers `functions` existants avec des noms identiques, vous devez donc choisir des noms de fichiers uniques pour chaque fichier que vous ajoutez manuellement. De plus, l'adaptateur ne videra jamais le dossier `functions` des fichiers obsolètes, vous devez donc nettoyer le dossier manuellement lorsque vous supprimez des pages.
+
+```diff lang="ts"
+  // astro.config.mjs
+  import {defineConfig} from "astro/config";
+  import cloudflare from '@astrojs/cloudflare';
+
+  export default defineConfig({
+    adapter: cloudflare({
+      mode: 'directory',
++     functionPerRoute: true
+    })
+  })
+```
+
+Notez que cet adaptateur ne prend pas en charge l'utilisation de [Cloudflare Pages Middleware](https://developers.cloudflare.com/pages/platform/functions/middleware/). Astro intègre le [Astro middleware](/fr/guides/middleware/) dans chaque page.
+
+### routes.strategy
+
+`routes.strategy: "auto" | "include" | "exclude"`
+
+Par défaut `"auto"`
+
+Détermine comment `routes.json` sera généré si aucun [`_routes.json`](#_routesjson-personnalisé) personnalisé n'est fourni.
+
+Il y a trois options disponibles :
+
+*   **`"auto"` (par défaut):** Sélectionne automatiquement la stratégie qui génère le moins d'entrées. Cela devrait presque toujours être suffisant, donc choisissez cette option à moins que vous n'ayez une raison spécifique de ne pas le faire.
+
+*   **`include`:** Les pages et les points de terminaison (endpoints) qui ne sont pas pré-rendus sont listés en tant qu'entrées `include` indiquant à Cloudflare d'invoquer ces routes en tant que fonctions. Les entrées `exclude` ne sont utilisées que pour résoudre les conflits. C'est généralement la meilleure stratégie lorsque votre site web a principalement des pages statiques et seulement quelques pages dynamiques ou points de terminaison.
+
+    Exemple: Pour `src/pages/index.astro` (statique), `src/pages/company.astro` (statique), `src/pages/users/faq.astro` (statique) et `/src/pages/users/[id].astro` (SSR) cela produira les `_routes.json` suivantes:
+
+    ```json
+    {
+      "version": 1,
+      "include": [
+        "/_image", // Point de Terminaison pour les images
```

Comment:
Either we keep the capitalize syntax. Or, we put all of theses in "normal" text format
```suggestion
        "/_image", // Point de terminaison pour les images
```